src/components/Auth/LoginForm.js

```diff
@@ -57,7 +59,9 @@ const LoginForm = () => {
 
   const submitHandler = (e) => {
     e.preventDefault();
-    dispatch(login({ account, password }));
+    const deviceToken = guid();
+    removeCookie('device_token');
+    dispatch(login({ deviceToken, account, password }));
```

Comment:
재사용 되지 않는다면 굳이 식별자로 선언할 필요가 있나 싶네요...!

```suggestion
    removeCookie('device_token');
    dispatch(login({ deviceToken: guid(), account, password }));
```

Reply:
앗 이 부분은 생각못했습니다. 알맞게 수정하겠습니다

Reply:
d75a572 회원 로그인에 대한 토큰 저장은 위와 같이 수정했습니다. 비회원 로그인에서는 deviceToken 값을 쿠키에 저장해야되서 일단은 그대로 두었습니다